x-pack/legacy/plugins/console_extensions/spec/overrides/ml.evaluate_data_frame.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
@dimitris-athanasiou I couldn't find docs for this API online. Do they live somewhere?

Reply:
No. We're working on the docs, they should be added this week.